src/test/java/uk/gov/companieshouse/api/testdata/service/impl/UserServiceImplTest.java

```diff
@@ -302,4 +302,64 @@ void testUpdateUserWithOneLoginUserNotFound() {
         verify(userRepository, times(1)).findById(userId);
         verify(userRepository, never()).save(any(User.class));
     }
+
+    @Test
+    void testCreateUserWithIsAdminTrue() throws DataException {
+        UserSpec userSpec = new UserSpec();
+        userSpec.setEmail("admin@hello.com");
```

Comment:
why are we setting  these admin@hello.com ??  we definitely don't want passwords in here either.  This should all be passed from the client ?